tavla/app/(admin)/organizations/components/DefaultColumns/index.tsx

```diff
@@ -28,31 +28,22 @@ function DefaultColumns({
     const { addToast } = useToast()
     const [open, setIsOpen] = useState(false)
 
-    const submit = async (
-        prevState: TFormFeedback | undefined,
+    const handleSaveColumns = async (
+        state: TFormFeedback | undefined,
         data: FormData,
     ) => {
+        if (!oid) return
         const columns = data.getAll('columns') as TColumn[]
-
-        if (!oid) {
-            return getFormFeedbackForError('auth/operation-not-allowed')
-        }
-
-        if (columns.length === 0) {
-            return getFormFeedbackForError('organization/invalid-columns')
-        }
-
-        saveColumns(oid, columns)
-        addToast('Kolonner lagret!')
+        const result = await saveColumnsAction(state, oid, columns)
+        fireToastFeedback(addToast, result, 'Kolonner lagret!')
+        return result
```

Comment:
Her trenger vi kanskje ikke en toast som sier "du trenger minst en kolonne"? Det for vi i alert-boksen under knappen
![image](https://github.com/user-attachments/assets/06ee0ec4-1e16-4f77-afe7-2f77ba29456e)

Reply:
Enig!